api/generated/v2/types.go

```diff
@@ -605,6 +611,9 @@ type TransactionApplication struct {
 	// \[apsu\] Logic executed for application transactions with on-completion set to "clear". It can read and write global state for the application, as well as account-specific local state. Clear state programs cannot reject the transaction.
 	ClearStateProgram *[]byte `json:"clear-state-program,omitempty"`
 
+	// \[epp\] specifies the additional app program len requested in pages.
+	ExtraProgramPages *uint64 `json:"extra-program-pages,omitempty"`
```

Comment:
We need to set this field in `api/converter_utils.go`.

Reply:
nice catch

Reply:
Thanks! Could you also add a test for `api.LookupTransaction()`?